common/src/main/scala/org/apache/celeborn/common/util/CelebornHadoopUtils.scala

```diff
@@ -17,13 +17,29 @@
 
 package org.apache.celeborn.common.util
 
+import java.util.concurrent.atomic.AtomicBoolean
+
 import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, Path}
 
 import org.apache.celeborn.common.CelebornConf
+import org.apache.celeborn.common.internal.Logging
 
-object CelebornHadoopUtils {
+object CelebornHadoopUtils extends Logging {
+  var logFlag = new AtomicBoolean(false)
   private[celeborn] def newConfiguration(conf: CelebornConf): Configuration = {
     val hadoopConf = new Configuration()
+    if (!conf.hdfsDir.isEmpty) {
+      val path = new Path(conf.hdfsDir)
+      val scheme = path.toUri.getScheme
+      val disableCacheName = String.format("fs.%s.impl.disable.cache", scheme)
+      hadoopConf.set("dfs.replication", "2")
+      hadoopConf.set(disableCacheName, "false")
+      if (logFlag.compareAndSet(false, true)) {
+        logInfo(
+          s"Celeborn overrides some HDFS settings defined in Hadoop configuration files, including '$disableCacheName=false' and 'dfs.replication=2'. It can be overridden again in Celeborn configuration with the additional prefix 'celeborn.hadoop.', e.g. 'celeborn.hadoop.dfs.replication=3'")
```

Comment:
does it violate the scalafmt? it's too long ...

Reply:
maxColumn = 100
It didn't violate the scalafmt.

Reply:
That's strange. I'll reformat this string.